src/containers/deleteEvent/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useHistory, useParams } from 'react-router-dom';
+import { Button, Typography } from 'antd';
+import { ContentContainer } from '../../components';
+import { Routes } from '../../App';
+import styled from 'styled-components';
+import { LinkButton } from '../../components/LinkButton';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import protectedApiClient from '../../api/protectedApiClient';
+const { Title } = Typography;
+
+const StyledButton = styled(Button)`
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  background-color: #2d870d;
```

Comment:
🔧 Theres a bunch of colors here that really should be constants defined in `utils/colors.ts`